socket_xdp_unspecified.go

```diff
@@ -0,0 +1,11 @@
+// +build !linux
```

Comment:
nit: add both directives here as well ?

Reply:
Thank you, it looks more consistent indeed. Second directive added.